shell/scripts/publish-shell.sh

```diff
@@ -45,17 +43,17 @@ popd >/dev/null
 
 echo "Publishing version: $PKG_VERSION"
 
-cp -R ${SHELL_DIR}/creators/app ${PKG_DIST}
-cp -R ${SHELL_DIR}/creators/pkg ${PKG_DIST}
-cp -R ${SHELL_DIR}/creators/update ${PKG_DIST}
+cp -R ${SHELL_DIR}/creators/extension ${PKG_DIST}
 
-sed -i.bak -e "s/\"0.0.0/"\"$PKG_VERSION"/g" ${PKG_DIST}/app/package.json
-sed -i.bak -e "s/\"0.0.0/"\"$PKG_VERSION"/g" ${PKG_DIST}/pkg/package.json
-sed -i.bak -e "s/\"0.0.0/"\"$PKG_VERSION"/g" ${PKG_DIST}/update/package.json
+sed -i.bak -e "s/\"0.0.0/"\"$PKG_VERSION"/g" ${PKG_DIST}/extension/package.json
+sed -i.bak -e "s/\"0.0.0/"\"$PKG_VERSION"/g" ${PKG_DIST}/extension/app/package.json
+sed -i.bak -e "s/\"0.0.0/"\"$PKG_VERSION"/g" ${PKG_DIST}/extension/pkg/package.json
+sed -i.bak -e "s/\"0.0.0/"\"$PKG_VERSION"/g" ${PKG_DIST}/extension/update/package.json
```

Comment:
don't know if we really need to change the pkg version to `app`, `pkg` and `update` here, but it won't hurt I believe

Reply:
I believe so, yeah... They won't get published individually anymore. Only the `@rancher/extensions` one will be published now